README.md

```diff
@@ -31,3 +31,7 @@ Branches are available at http://preview.iiif.io/api/[branch_name]/api/index.htm
     * link from the API website to the main website (not relative to this repo) `[hyperlink text]({{ page.webprefix }}/end/point)`
     * External link `[anchor-text](http://example.com/end/point)`
     * Reference link `[text][link_name]` where link_name is expanded at the bottom of the page.
+
+## Contributions
+
+Contributions are welcome in the form of pull requests. By making such a pull request the contributor asserts that they have the right to license the contribution (of whatever form) such that content may be published under the CC-BY license by the IIIF Consortium, per our license statement on all specifications.
```

Comment:
new issues or pull requests?

Reply:
oh, then also have to change second sentence to "By making a pull request..."

Reply:
Made some more sweeping changes to clarify contribution vs general participation. Does this fix?